Lib/test/test_compile.py

```diff
@@ -752,6 +752,12 @@ def continue_in_while():
             self.assertEqual(None, opcodes[0].argval)
             self.assertEqual('RETURN_VALUE', opcodes[1].opname)
 
+    def test_big_dict_literal(self):
+        # See bpo-41531
+        dict_size = 2 * 0xFFFF
+        the_dict = "{" + ",".join(f"{x}:{x}" for x in tuple(range(dict_size))) + "}"
```

Comment:
I'm not curous: why tuple(range(dict_size)), instead of range(dict_size)?

Reply:
> I'm not curous: why tuple(range(dict_size)), instead of range(dict_size)?

No reason, I was constructing the dictionary in a different way and at that point I needed a tuple and not an iterator, will change